src/core/js/models/itemsModel.js

```diff
@@ -5,8 +5,13 @@ define([
 
     var ItemsModel = ComponentModel.extend({
 
+        reset: function(type) {
+            this.resetItems();
+            ComponentModel.prototype.reset.call(this, type);
+        },
```

Comment:
```
        reset: function(type, force) {
            this.resetItems();
            ComponentModel.prototype.reset.call(this, type, force);
        },
```
Sorry I missed out the second property last night.
This should also be added to the ItemsModel reset function.